accounts-db/src/accounts_index_storage.rs

```diff
@@ -64,34 +64,21 @@ impl BgThreads {
     ) -> Self {
         // stop signal used for THIS batch of bg threads
         let local_exit = Arc::new(AtomicBool::default());
-        let handles = Some(
```

Comment:
nit: Can we comment out the code instead of deleting it since we don't have a replacement right now? I think it'll also make updates easier when we rebase on `main`, i.e. what is a new change we have to pull in VS what are old changes we removed on purpose.

Reply:
I'll redo this so it's gated on a compile flag.